civictechprojects/static/css/projects.css

```diff
@@ -203,7 +203,6 @@ th, td {
       font-family: Helvetica;
       font-size: 14px;
       font-weight: 300;
-      text-align: left;
```

Comment:
The proper way to revert this change would be to revert it to 'text-align: right'